### PR TITLE
Prison Wardrobe Tweaks

### PR DESCRIPTION
**Pull request by liltenhead**
## About the PR
Adds Vox tank harnesses and passenger headsets to prisoner wardrobes. Vox need a way to hold tanks more comfortably, prisoners have a right to communications. 

## Why / Balance
Space law

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: Prisoner wardrobes now contain tank harnesses and passenger headsets.

### CODE REVIEW

**Comment by TheCrimsonJupiter** [comment]
These are very good and needed changes for prisoners!

**Comment by BoskiYourk** [comment]
Happy to see this! Good shit

File: Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml [hunk 1]
```diff
@@ -13,6 +13,10 @@
        - id: ClothingShoesColorBlack
          amount: 2
        - id: ClothingMaskMuzzle
+       - id: ClothingOuterVestTank
+         amount: 2
+       - id: ClothingHeadsetGrey
+         amount: 2
```
Comment (FluffMe) [inline Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml:19]:
```suggestion
       - id: ClothingOuterVestTank # Harmony. Required for Vox by space law
         amount: 2
       - id: ClothingHeadsetGrey # Harmony. Required by space law
         amount: 2
```

**Review by FluffMe — CHANGES_REQUESTED** [review]
Missing Harmoy comments in upstream yaml

**Review by FluffMe — APPROVED** [review]

**Pull request merged**